### PR TITLE
Release browser-sdk 3.0.0

### PR DESCRIPTION
**Pull request by Worvast**
This new version merge internal browser-sdk and open source browser-sdk to a new unique one.

Its revised by Activeboard team but we want one external eye

## Version 3.0.0

* Replaced `stream` for `streamFetch`, this method, to stream back query, returns data in arrays filled with multiple data objects that have been parsed in chunks. With this change we have noticeably improved the performance, and also made code more secure and efficient.

* Now is required to use a node version greater than 14.0.0 and npm version greater than 7.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 Change log.
 
+## Version 2.0.0
+
+* Replaced `stream` for `streamFetch`, this method, to stream back query, returns data in arrays filled with multiple data objects that have been parsed in chunks. With this change we have noticeably improved the performance, and also made code more secure and efficient.
+
+* Now is required to use a node version greater than 14.0.0 and npm version greater than 7.0.0
```
Comment (tripu) [inline CHANGELOG.md:9]:
```suggestion
* Now is required to use Node.js version &ge; 14.0.0 and npm version &ge; 7.0.0
```

**Review by tripu — CHANGES_REQUESTED** [review]
Now it is recommended to keep the lock file under version control, too.

I would remove [its entry from `.gitignore`](https://github.com/DevoInc/browser-sdk/blob/master/.gitignore#L7) and commit the file.

**Comment by Worvast** [comment]
> Now it is recommended to keep the lock file under version control, too.
> 
> I would remove [its entry from `.gitignore`](https://github.com/DevoInc/browser-sdk/blob/master/.gitignore#L7) and commit the file.

ITs removed because sure enter in conflict with the merge of the version 2.0 Antonio is merging. When PR #14 is merged, i update this branch with master and update again package-lock.json. That was i delete the lock, but not add it to gitignore

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@devo/browser-sdk",
-  "version": "1.3.3",
+  "name": "@devoinc/browser-sdk-dashboard",
+  "version": "3.0.0",
```
[the remark this answers is not in the document]
Comment (Worvast) [reply package.json:3]: Version v2 is the same-version update to node14 in a new @devoinc scope, in PR #14 

this version is breaking change, in the new scope too, but under version 3.0

The reason for this major is have a non-big-breaking-change version under the new scope, without a deprecate scope.

[the remark this answers is not in the document]
Comment (tripu) [reply package.json:3]: Mmmh I'm OK either way.

But both this PR and #14 are making the same changes (changing the scope, and bumping the major to 2). Just one of the two should be doing these changes, I think. It doesn't make sense to have both PRs doing that.

Also, you say this is changing the version to `2.0.0` both in the title and in the description of the PR. Either the PR is wrong, or the changes in the code are wrong. Right?

**Comment by tripu** [comment]
> > Now it is recommended to keep the lock file under version control, too.
> > I would remove [its entry from `.gitignore`](https://github.com/DevoInc/browser-sdk/blob/master/.gitignore#L7) and commit the file.
> 
> ITs removed because sure enter in conflict with the merge of the version 2.0 Antonio is merging. When PR #14 is merged, i update this branch with master and update again package-lock.json. That was i delete the lock, but not add it to gitignore

OK, I see #14 makes that change already :+1:

(Anyway, modern versions of npm usually can handle changes in the package lock quite well; no need to mess with it manually, nor to remove and recreate it.)

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Worvast) [reply package.json:3]: The problem was i create this, talk with Antonio, he create the other, determine this changes and i forgot change the title of the PR (And obviusly the branch, but not important)

When the other PR is merged and i update my branch with master this changes dispear, not a big deal here. This PR is created with the principa objetive of revise the big changes.

File: package-lock.json [hunk 3]
```diff
@@ -1,3187 +1,3158 @@
 {
   "name": "@devoinc/browser-sdk",
-  "version": "2.0.0",
-  "lockfileVersion": 2,
+  "version": "3.0.0",
+  "lockfileVersion": 1,
```
Comment (tripu) [inline package-lock.json:4]: Wrong file version here. You must have updated/recreated the lock with a version of npm < 6.

**Review by tripu — CHANGES_REQUESTED** [review]

**Review by tripu — APPROVED** [review]

**Review by apbarrero — APPROVED** [review]

**Pull request merged**